src/prefect/infrastructure/docker.py

```diff
@@ -277,9 +280,35 @@ async def run(
         exit_code = container.attrs["State"].get("ExitCode")
         return DockerContainerResult(
             status_code=exit_code if exit_code is not None else -1,
-            identifier=container.id,
+            identifier=container_pid,
         )
 
+    async def kill(self, infrastructure_pid: str, grace_seconds: int):
```

Comment:
Should grace seconds here also have default?

Reply:
Yes definitely, thanks for the catch!